src/DynamoCoreWpf/ViewModels/Watch3D/DynamoGeometryModel3D.cs

```diff
@@ -90,9 +97,9 @@ protected override RenderCore OnCreateRenderCore()
             return new DynamoGeometryMeshCore();
         }
 
-        protected override IRenderTechnique OnCreateRenderTechnique(IRenderHost host)
+        protected override IRenderTechnique OnCreateRenderTechnique(IEffectsManager effectsManager)
```

Comment:
Is this function signature change valid only for .net6 or .net framework as well?

Reply:
I updated HelixToolKit for .net framework too. In fact the update in .net 6 is almost the same but with different namespaces. If you go to the library repository you will notice that they change the namespace to .net and make some conditions. So, the change is valid for both frameworks